doc/sphinx/reactors.rst

```diff
@@ -1,74 +1,38 @@
+    - Heat transfer: An arbitrary heat transfer rate can be defined to cross the boundaries of the reactor.
+    - Mass transfer: The reactor can have multiple inlets and outlets. For the inlets, arbitrary states can be defined. Through the outlets, fluid with the current state of the reactor exits the reactor.
+    - Surface interaction: One or multiple walls can influence the chemical reactions in the reactor. This is not just restricted to catalytic reactions, but mass transfer between the surface and the fluid can also be modeled.
 
-    \dot{m}_{wall} = \sum_k \dot{m}_{k,wall}
+All of these interactions do not have to be constant, but can vary as a function of time or state. For example, heat transfer can be described as a function of the temperature difference between the reactor and the environment, or the wall movement can be modeled depending on the pressure difference. Typically, interactions of the reactor with the environment are defined on one or multiple *walls*, *inlets*, and *outlets*.
 
-The total rate of heat transfer through all walls is:
 
-.. math::
+In addition to single reactors, Cantera is also able to interconnect reactors into a *Reactor Network*. Each reactor in a network may be connected so that the contents of one reactor flow into another. Reactors may also be in contact with one another or the environment via walls which move or conduct heat.
 
-    \dot{Q} = \sum_w f_w \dot{Q}_w
 
-General Reactor
-===============
+Governing Equations for Single Reactors
+=======================================
 
 The state variables for Cantera's general reactor model are
 
-    - `m`, the mass of the reactor's contents
-    - `V`, the reactor volume
-    - `U`, the total internal energy of the reactors contents
-    - `Y_k`, the mass fractions for each species
-
-Reactor Volume
---------------
-
-The reactor volume changes as a function of time due to the motion of one or
-more walls:
-
-.. math::
-
-    \frac{dV}{dt} = \sum_w f_w A_w v_w(t)
-
-where `f_w = \pm 1` indicates the facing of the wall, `A_w` is the surface
-area of the wall, and `v_w(t)` is the velocity of the wall as a function of
-time.
+    - `m`, the mass of the reactor's contents (in kg)
+    - `Y_k`, the mass fractions for each species (dimensionless)
+    - `V`, the reactor volume (in m\ :sup:`3`) (not a state variable for *Constant Pressure Reactor* and *Ideal Gas Constant Pressure Reactor*)
+    - A state variable describing the energy of the system, depending on the configuration (see `Energy Conservation`_ for further explanation):
+        - General *Reactor*: `U`, the total internal energy of the reactors contents (in J)
+        - *Constant Pressure Reactor*: `H`, the total enthalpy of the reactors contents (in J)
+        - *Ideal Gas Reactor* and *Ideal Gas Constant Pressure Reactor*: `T`, the temperature (in K)
```

Comment:
I think it's better to list these in the order m, V, U/H/T, Yk, to be consistent with the order that the variables are in the state vector (which ends up exposed to the user when working with sensitivity coefficients).